apps/gk_species_radiation.c

```diff
@@ -44,25 +46,57 @@ gk_species_radiation_init(struct gkyl_gyrokinetic_app *app, struct gk_species *s
     gkyl_cart_modal_gkhybrid(&surf_mu_basis, cdim, 1);   
   }
 
+  // Updater to compute drag coefficients.
+  rad->calc_gk_rad_vars = gkyl_dg_calc_gk_rad_vars_new(&s->grid, &app->confBasis,
+	&app->basis, s->info.charge, s->info.mass, app->gk_geom, s->vel_map, app->use_gpu);
+
   // Fitting parameters
-  double a[26], alpha[26], beta[26], gamma[26], v0[26];
-  struct all_radiation_states *rad_data=gkyl_read_rad_fit_params();
+  double a[GKYL_MAX_RAD_DENSITIES], alpha[GKYL_MAX_RAD_DENSITIES], beta[GKYL_MAX_RAD_DENSITIES],
```

Comment:
This feedback may be a bit delayed, but please consider changing this n0 and this ne variables to something more descriptive and easier to work with, given that:
1. Single letter variables should not exist (except a temporary variable in a small piece of code, like the for the counter of a small loop).
2. ne is pretty generic. It could mean the electron density, but that is x and t dependent.
Perhaps all these fit parameters should be something like radfit_a for a, etc.